ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -387,6 +392,20 @@ def test_source_ipa_roles_check_crlmanager(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == "crl_manager"
             assert check["kw"]["crlgen_enabled"] is False
+        self.master.run_command(["ipa-crlgen-manage", "enable"])
```

Comment:
I would put this command in a `finally:` block to make sure it is called even if the test fails.